packages/next/src/generators/library/schema.json

```diff
@@ -72,9 +72,8 @@
     },
     "unitTestRunner": {
       "type": "string",
-      "enum": ["jest", "none"],
-      "description": "Test runner to use for unit tests.",
-      "default": "jest"
+      "enum": ["jest", "vitest", "none"],
```

Comment:
This seems unrelated?

Reply:
Making sure we can use Vitest everywhere basically.